autoload/unite/sources/tselect.vim

```diff
@@ -41,16 +41,27 @@ function! s:read_line_fmt(tag)
 	let l:line = system("sed '" . a:tag.cmd . "q;d' " . a:tag.filename)
 	let l:line = substitute(l:line, '^\s\+', '', '')
 	let l:line = substitute(l:line, "\n", "", "")
-	return '['. a:tag.kind . '] ' . a:tag.filename . ":\t" . l:line
+    return s:output_for_tag(a:tag, l:line)
```

Comment:
Please use tab instead of 4 spaces.

Reply:
Sorry, I caught that before opening the PR but failed to actually push it to the right branch.